src/telemetry.cpp

```diff
@@ -29,10 +29,32 @@
 constexpr int CONNECTION_STABILITY_MSEC = 45000;
 
 constexpr const uint32_t VPNSESSION_PING_TIMER_SEC = 3 * 60 * 60;  // 3 hours
-constexpr const uint32_t VPNSESSION_PING_TIMER_DEBUG_SEC = 120;
 
 namespace {
 Logger logger("Telemetry");
+
+/**
+ * @brief Get a stringified State and return the equivalent telemetry screen
+ * name.
+ *
+ * The state name is camelCase and has the "State" prefix,
+ * while the telemetry screen identifier is snake_case and has no prefix.
+ */
+QString stateToTelemetryScreenName(const QString& input) {
```

Comment:
Hm, actually. As I dig more into the telemetry design I realize this implementation is incorrect, so I'll change it and also remove this function altogether. 

We should not be recording an impression on every state change. Impression metrics are meant to be recorded as a user moves on to a new screen. Although this looks clever, it will not be what Yeonjoo and Santiago expect. We need to record these impression metrics from QML as each screen is loaded. State changes don't map 1:1 to new screens :/